client/src/components/App.jsx

```diff
@@ -15,12 +14,13 @@ export class App extends Component {
   }
 
   componentDidMount() {
-    const { fetchEntries, userId } = this.props;
-    fetchEntries(userId);
+    const { dispatchGetEntries, userId } = this.props;
+    dispatchGetEntries(userId);
```

Comment:
You can use this.prop.dispatchGetEntries ... don't think we need to define the variables on line 17